scheduler/replica_scheduler_test.go

```diff
@@ -92,11 +92,11 @@ func newDaemonPod(phase v1.PodPhase, name, namespace, nodeID, podIP string) *v1.
 	}
 }
 
-func newNode(name, namespace string, allowScheduling bool, nodeState types.NodeState) *longhorn.Node {
+func newNode(name, namespace string, allowScheduling, diskAllowScheduling bool, nodeState types.NodeState) *longhorn.Node {
```

Comment:
We can separate disk creation into `newDisk`, and test node with multiple disks. Here it seems only covered single disk cases. It's better to cover 0 disk and multiple disks with different allowScheduling set as well